packages/headless-components/events/src/react/EventList.tsx

```diff
@@ -322,23 +322,20 @@ export interface CategoryFilterProps {
  * ```
  */
 export const CategoryFilter = (props: CategoryFilterProps): React.ReactNode => {
+  const { className, allCategoriesLabel } = props;
```

Comment:
`children` is missing - `const { children, className, allCategoriesLabel } = props;`